### DISCUSSION TITLE
renamerOnUpdate: Path

### DISCUSSION THREAD

**Discussion by Asuryan34** [stashapp/stash#4191] [Q&A]
Hello.

I need a bit help with my path.

I have my stash installed on an NAS (Docker)
my videos are on "Stash\data\videos\" (Library set on "Stash\data\")

Since I just started using stash, my "videos" folder is a totals mess. Full of subfolders in subfolders, etc.

I can do simple "sort" like. If you have this tag, go to this folder.

However, it's not really what I would prefer.

The example in the config.py says: 

trigger with a specific tag
"tagname": "path"
 ex: "plugin_move": r"E:\Movies\R18\$studio_hierarchy"

Would it be possible to have multiple tag ?

If file has 123, 789, 456 as tags,  file moves to .../123/789/456
If file has 123, 102 as tags, file moves to .../123/102

etc.

It would allow a better organization

**Comment by scruffynerf** [comment]
Renamer on update is not part of Stash proper but a plugin in the Community Script repo.  You should open an issue there not here.
And while I appreciate your desire to add features suggestions, the original author hasn't been around, and we're likely going to update and rewrite it.  So contribute the suggestion as a Feature Request not just an issue on that repo so we can track it
